.vscode/launch.json

```diff
@@ -7,15 +7,15 @@
 		{
 			"type": "node",
 			"request": "launch",
-			"cwd": "${workspaceRoot}",
+			"cwd": "${workspaceRoot}/scratch/app1",
```

Comment:
revert this file

Reply:
ℹ️ `git update-index --assume-unchanged .vscode/launch.json`